src/main/java/io/nats/client/Options.java

```diff
@@ -230,11 +227,16 @@ public class Options {
      * {@link Builder#dataPortType(String) dataPortType}.
      */
     public static final String PROP_DATA_PORT_TYPE = PFX + "dataport.type";
+    /**
+     * Property used to configure a builder from a Properties object. {@value}, see
+     * {@link Builder#natsLogger(NatsLogger) natsLogger}.
+     */
+    public static final String PROP_ERROR_LISTENER = PFX + "logger";
     /**
      * Property used to configure a builder from a Properties object. {@value}, see
      * {@link Builder#errorListener(ErrorListener) errorListener}.
      */
-    public static final String PROP_ERROR_LISTENER = PFX + "callback.error";
+    public static final String PROP_LOGGER = PFX + "callback.error";

```

Comment:
Does not seem like the correct property name.

Reply:
:-) woah must have been too late... Fixed.